specs/ScrollingControls/ScrollPresenter.md

```diff
@@ -0,0 +1,2462 @@
+        Microsoft.UI.Xaml.Controls.ScrollingZoomOptions options);
+    Int32 AddZoomVelocity(
+        Single zoomFactorVelocity,
+        Windows.Foundation.IReference<Windows.Foundation.Numerics.Vector2> centerPoint,
+        Windows.Foundation.IReference<Single> inertiaDecayRate);
+
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, Object> ExtentChanged;
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, Object> StateChanged;
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, Object> ViewChanged;
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, Microsoft.UI.Xaml.Controls.ScrollingScrollAnimationStartingEventArgs> ScrollAnimationStarting;
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, Microsoft.UI.Xaml.Controls.ScrollingZoomAnimationStartingEventArgs> ZoomAnimationStarting;
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, Microsoft.UI.Xaml.Controls.ScrollingScrollCompletedEventArgs> ScrollCompleted;
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, Microsoft.UI.Xaml.Controls.ScrollingZoomCompletedEventArgs> ZoomCompleted;
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, Microsoft.UI.Xaml.Controls.ScrollingBringingIntoViewEventArgs> BringingIntoView;
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, Microsoft.UI.Xaml.Controls.ScrollingAnchorRequestedEventArgs> AnchorRequested;
+    event Windows.Foundation.TypedEventHandler<ScrollPresenter, MU_XCP_NAMESPACE.ScrollingViewChangeRequestedEventArgs> ViewChangeRequested;
+
+    static Microsoft.UI.Xaml.DependencyProperty BackgroundProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty ContentProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty ContentOrientationProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty HorizontalScrollChainModeProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty VerticalScrollChainModeProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty HorizontalScrollRailModeProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty VerticalScrollRailModeProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty HorizontalScrollModeProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty VerticalScrollModeProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty ZoomChainModeProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty ZoomModeProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty IgnoredInputKindsProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty MinZoomFactorProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty MaxZoomFactorProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty HorizontalAnchorRatioProperty { get; };
+    static Microsoft.UI.Xaml.DependencyProperty VerticalAnchorRatioProperty { get; };
+}
+```
+
+# Sample IScrollController implementations
```

Comment:
These last samples are great, but long and I don't want reviewers to feel compelled to provide feedback on this much content. Also I think this would go into the samples repo rather than the docs. Could you keep them but put them in the appendix?